README.md

```diff
@@ -2,21 +2,37 @@
 
 ## install
 
-> git clone --recursive git@github.com:talis/citeproc-node.git
+```bash
+$ git clone --recursive git@github.com:talis/citeproc-node.git
+$ npm rebuild // rebuild step probably isn't necessary, but won't hurt
```

Comment:
if the version of node is fixed and you know it works against that then this is probably overkill (and slower) only really makes sense to do this if the version of node might be changing.